contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -225,6 +227,10 @@ func NewSchedulerServer() *SchedulerServer {
 		// when kube-proxy is running in a non-root netns (init_net); setting this to a non-zero value will
 		// impact connection tracking for the entire host on which kube-proxy is running. xref (k8s#19182)
 		conntrackMax: 0,
+
+		// cadvisor housekeeping related config
+		housekeepingInterval:       1 * time.Second,
```

Comment:
when I compiled your branch and looked at the flags of `km scheduler` I see
this:
```
      --housekeeping-interval=1s: Interval between container housekeepings
      --housekeeping_interval=10s: Interval between container housekeepings
```

the 1s belonged to the flag that you created, the 10s belonged to the
cadvisor global flag.


On Thu, Feb 4, 2016 at 4:19 PM, Lin Sun <notifications@github.com> wrote:

> In contrib/mesos/pkg/scheduler/service/service.go
> <https://github.com/kubernetes/kubernetes/pull/20331#discussion_r51938799>
> :
>
> > @@ -225,6 +227,10 @@ func NewSchedulerServer() *SchedulerServer {
> >  		// when kube-proxy is running in a non-root netns (init_net); setting this to a non-zero value will
> >  		// impact connection tracking for the entire host on which kube-proxy is running. xref (k8s#19182)
> >  		conntrackMax: 0,
> > +
> > +		// cadvisor housekeeping related config
> > +		housekeepingInterval:       1 * time.Second,
>
> Hmm, I was reading it from cadvisor/docs/runtime_options.md, it had:
>
> --housekeeping_interval=1s: Interval between container housekeepings
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/20331/files#r51938799>.
>